go/enclave/enclave_admin_service.go

```diff
@@ -119,17 +120,15 @@ func NewEnclaveAdminAPI(config *enclaveconfig.EnclaveConfig, storage storage.Sto
 	if eas.isBackupSequencer(context.Background()) || eas.isActiveSequencer(context.Background()) {
 		mempool.SetValidateMode(false)
 	}
-	if eas.isActiveSequencer(context.Background()) {
```

Comment:
Sequencer is never going to be active when it first starts up so we don't check this in constructor.